packages/driver/src/cypress/cy.ts

```diff
@@ -535,12 +553,12 @@ export class $Cy implements ITimeouts, IStability, IAssertions, IRetries, IJQuer
           e = onpl(e)
         }
 
-        // and now reject with it
-        const r = this.state('reject')
+        Cypress.action('cy:cross:domain:failure', e)
```

Comment:
Can we be sure this was a cross:domain failure?

Reply:
Yes, outside of errors from a user's `window:load` handler, which is handled above, the only reason the code in the `try` will fail is with a cross-origin failure because we attempt to set properties on window that is cross-origin.